### PR TITLE
Hmv styles

### PR DESCRIPTION
**Pull request by Jbur43**
These edits came from these comments:

- Archetype description is not vertically centered
- Archetype box could have more padding on the top and bottom
- Dimension names should be a bit smaller so that “Emotional Stability” text fits better
- For the breakdown section, the description text is being stopped short and can flow out closer the the right edge. Opposite to that the  section for Benefits and Growth should have double the padding on the side. It is too close the the edges

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -3238,6 +3238,17 @@
           "requires": {
             "ansi-regex": "^4.1.0"
           }
+        },
+        "write-file-atomic": {
```
Comment (tomprats) [inline package-lock.json]: What's up with these package-lock changes?

**Comment by tomprats** [comment]
Aside from the package-lock change, it looks good. But might be good just to push your branch to staging and have Josh take a look so there isn't as much of a back and forth

File: package-lock.json (hunk 1 above)
Comment (Jbur43) [reply package-lock.json]: Looks like its required by "@jest/transform"

Comment (Jbur43) [reply package-lock.json]: Nothing productive. I updated it.

**Comment by Jbur43** [comment]
Yeah I agree

**Pull request merged**